contrib/runners/announcement_runner/announcement_runner/runner.yaml

```diff
@@ -16,3 +16,6 @@
       maxLength: 255
       minLength: 1
       type: string
+  output_schema:
+    unmodeled:
+      type: boolean
```

Comment:
Why do we need to have this unmodeled property here?  If there's no output schema or if output schema is empty, isn't it unmodeled by default?